src/test/resources/features/PostalCode.feature

```diff
@@ -0,0 +1,63 @@
+@PostalCode @smoke
+Feature: Get location information of a city using country and postal code
+
+  Scenario Outline: Get a successful response when sending a valid postal code and a valid country
+    Given The user has access to the zippopotamus endpoint
+    When The user performs a GET request with a <country> and <postalCode>
+    Then The response status code should be 200
+    And The response body should contains <postalCode> as postal code
+
+    Examples:
+      | country | postalCode |
+      | "us"    | "90210"    |
+      | "hr"    | "10000"    |
+      | "at"    | "1010"     |
+      | "ar"    | "3500"     |
+
+  Scenario Outline: Get a unsuccessful response when sending a invalid postal code and a valid country
+    Given The user has access to the zippopotamus endpoint
+    When The user performs a GET request with a <invalidCountry> and <postalCode>
+    Then The response status code should be 404
+
+    Examples:
+      | invalidCountry | postalCode |
+      | "us1"          | "90210"    |
+      | "1us"          | "90210"    |
+      | "uss"          | "90210"    |
+      | "uus"          | "90210"    |
+      | "hr."          | "10000"    |
+      | ".at"          | "1010"     |
+      | "aarr"         | "3500"     |
+
```

Comment:
Please add the "" (empty) parameter for the country, that was deleted in the previous commits, as the last example here.

Reply:
Done